spec.go

```diff
@@ -0,0 +1,11 @@
+package flanneltpr
+
+import (
+	"github.com/giantswarm/flanneltpr/host"
+	"github.com/giantswarm/flanneltpr/network"
+)
+
+type Spec struct {
+	Host    host.Host       `json:"host" yaml:"host"`
+	Network network.Network `json:"network" yaml:"network"`
```

Comment:
I'm not sure how to obtain cluster ID -- it's needed to observe if namespace is gone.